.rhcicd/pr_check.sh

```diff
@@ -31,8 +31,17 @@ source $CICD_ROOT/cji_smoke_test.sh
 
 # Until test results produce a junit XML file, create a dummy result file so Jenkins will pass
 mkdir -p $WORKSPACE/artifacts
-cat << EOF > ${WORKSPACE}/artifacts/junit-dummy.xml
-<testsuite tests="1">
-    <testcase classname="dummy" name="dummytest"/>
-</testsuite>
-EOF
+
+# adding floorist tests.
+<<<<<<< Updated upstream
+export COMPONENT_NAME="policies-engine"
+=======
+>>>>>>> Stashed changes
```

Comment:
This looks like an unresolved conflict.

Reply:
Thanks for pointing this out. I am not sure what happened to get this code inserted. Something I have not seen before with git stash.

Reply:
I am guessing I stashed something that I then changed upstream before I popped it and added to commit.